gateway-js/src/index.ts

```diff
@@ -970,7 +956,8 @@ export class ApolloGateway implements GraphQLService {
     }
 
     if (!queryPlan) {
-      queryPlan = buildQueryPlan(operationContext, {
+      // TODO: Can we be sure the query planner has been initialized here?
```

Comment:
Agreed, but this PR doesn't change the current behavior. We were already relying on the query planner being initialized at this point. I'm not sure how/if that is guaranteed, so that's why I left a comment to get back to this. (I suspect we'll want to take advantage of the existing `GatewayState` type to take a more principled approach to this.)

Reply:
Opened #631